core/spec/models/spree/order/totals_spec.rb

```diff
@@ -6,7 +6,7 @@ module Spree
     let(:shirt) { create(:variant) }
 
     context "adds item to cart and activates promo" do
-      let(:promotion) { Promotion.create name: 'Huhu' }
+      let(:promotion) { Promotion.create name: 'Huhu', apply_automatically: true }
```

Comment:
It does seem like we should be using a factory here but that's not the fault of this PR.  `create(:promotion, apply_automatically: true)` seems easy enough to me w/o a trait or factory.